source/backend/utils/Information.cpp

```diff
@@ -65,10 +65,10 @@ const char* carla_get_complete_license_text()
         "<li>LADSPA plugin support</li>"
         "<li>DSSI plugin support</li>"
         "<li>LV2 plugin support</li>"
-#if defined(USING_JUCE) && (defined(CARLA_OS_MAC) || defined(CARLA_OS_WIN))
+#if defined(USING_JUCE) && (defined(CARLA_OS_MAC) || defined(CARLA_OS_WIN) || defined(CARLA_OS_LINUX))
```

Comment:
we can remove the OS check here and only check if juce is being used. since it only works for these OSes anyway